### DISCUSSION TITLE
SSL LetEncrypt not auto redirect to https

### DISCUSSION THREAD

**Discussion by tracer8** [requarks/wiki#3849] [Error / Bug Report] [answered]
Hi,
I'm using Letencrypt for SSL. I must add https to my url to make ssl working, if not, website will not return any response. It is not automatic redirect to https.
And when setup, i must set my server HTTP port to 80, to make Letencrypt init working, if not, it will trigger fail.

**Comment by Phil-hostwiki** [comment, marked as answer]
1. Goto Administration Area.
2. Click on the SSL menu.
3. Turn On HTTP to HTTPS redirection.

**Reply by tracer8** [reply]
Oh, it already on.
But it is working now, look like SSL from Letencrypt need more time to making HTTPS auto redirection.
Thanks!
